wazo_confd/plugins/event_handlers/service.py

```diff
@@ -116,6 +116,23 @@ def generate_sip_templates(self, tenant):
             tenant.webrtc_sip_template_uuid,
         )
 
+        webrtc_video_config = {
+            'label': 'webrtc_video',
+            'template': True,
+            'tenant_uuid': tenant.uuid,
+            'transport': transport_wss,
```

Comment:
```suggestion
            'transport': None,
```
It already inherit from `webrtc_template`